docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -4,8 +4,67 @@ sidebar_position: 1
 
 # Creating Service Binding
 
-In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
-- Exposing binding-metadatas by default
-If it's not the case, please refer to the following section(#)
+In this section, we consider that the service you want to connect to is already
+exposing binding metadata by either:
 
+1. Provisioned Service
+2. Direct Secret Reference
+3. Secret Generated  through CRD/CR annotations
+4. Secret Generated through OLM Descriptors
+5. Detect Bindings through owned resources
+
+If it's not the case, please refer to the "exposing binding data" section.
+
+You also required the application ready to create the ServiceBinding resource.
+
+There are two APIs supported by the Service Binding Operator.  Both has
+`ServiceBinding` as the same name for the kind.  But the API group is different.
+
+1. binding.operators.coreos.com - This API will be removed in future.
```

Comment:
Let's not talk about it in the userdoc, cc @slemeur

Reply:
Do you mean that we only need to document spec API?

Reply:
ah, sorry - misunderstanding - I would not mention here at all any deprecation plans. We should describe both apis and give where needed details what is different.

Reply:
Done. dc17740535825e50ffe82736d6b9226ac6bcb631